dev-util/pycharm-community/pycharm-community-2024.3.5.ebuild

```diff
@@ -128,6 +135,38 @@ src_prepare() {
 		-e "\$a#-----------------------------------------------------------------------" \
 		-e "\$aide.no.platform.update=Gentoo" bin/idea.properties
 
+	# excepting files that should be kept for remote plugins
+	# but removing the ones that are not needed for the current architecture
```

Comment:
Nitpick - this comment is a bit misleading, it implies removing files but it's about removing debug symbols if I understand this right.  Not important to hold merging, but keep in mind for next time.